### DISCUSSION TITLE
Integrated game guides

### DISCUSSION THREAD

**Discussion by suspect15** [RetroAchievements/RAWeb#1051] [Ideas]
I would like to suggest making it possible to add a link to an RAGuide on a set's page.  Similar to how a video can be embedded in an achievement's page, easy access to the RAGuide may make them more frequently used.  This also might promote more interest in creating them as well.

**Comment by Tsearo** [comment]
Seems like a nice idea, could add a database field for it to GameData, a text field in the game page dev dropdown to enter a valid  https://github.com/RetroAchievements/guides/wiki/ url and then display a section in More Info if it exists like this
![image](https://user-images.githubusercontent.com/4047771/179362896-ef7653fa-947f-4b22-b935-81165dd05e8b.png)

Could also display an indicator on other pages like game list for games that have guides and possibly filters

**Reply by suspect15** [reply]
That looks great!  Exactly what I had in mind.

**Reply by Krylan** [reply]
I had the same idea for some time, and shared it few days ago with the group (but outside Github). I also think that "More Info" section in sidebar is the best place to put the link to the guide. The only thing I had different in my mind, is using other emoji (the book 📕, instead of writing ✍️).
These links will make discovering guides for games much easier.

**Reply by meleu** [reply]
+1 for using the book rather than the writing hand.

Also, I'm gonna create a site for the guides so we can have a nicer view than the github wiki (maybe something like RADocs).

**Comment by ScottFromDerby** [comment]
Agreed, this is 100% a great idea. Would love to have a custom section of the site dedicated to hosting these bespoke guides

**Comment by televandalist** [comment]
Personally, I'd rather guides become an on-site thing so we can use the [game=#####] and [ach=######] tags. Maybe eventually have a checklist to show what you have and haven't earned in the context of the guide.

**Comment by televandalist** [comment]
(It would be a lot better than hotlinking and manually typing out achievement info that would need to be edited in the guide wiki if an achievement gets revised)

**Comment by Zenjir0** [comment]
Just adding to the discussion. 

With popular third-party sites in [flux](https://alternativeto.net/news/2022/10/fandom-has-purchased-gamespot-metacritic-tv-guide-and-more-from-red-ventures/), it would not hurt for RA to have their own dedicated FAQs/Walkthroughs that users can submit. 

I like Tsearo's thinking of where it should be located. 

I think where my thinking differs is that RAGuides is more geared towards completing the set. And I am thinking more on expanding it to the lines of having user submitted walkthroughs of the game that are agnostic of the achievement set(s), along with guides for achievements as well.

**Comment by Krylan** [comment]
We should think thoroughfully about the way, how it should be implemented. We have multiple ways of doing it, each one having its trade-offs. Right now, we have probably only a single guide for each of the games, but I think we should support multiple of them especially, that each of them could focus on different things (sometimes out there are published informational guides with bestiaries, side-quests etc.).

We should also think, where those guides should be hosted. Right now, they are on the Github, which has some advantages, like whole editing and collaboration utillities (with full version history). On the other hand, it requires Github account from people who want to contribute to guides. Also, we have limited capacity of linking achievements. In this case, we would also need to decide, if guides on game page should be just a link to GH wiki page, or the raw content from GH should be displayed in RA template (then we would need to convert markdown notation to HTML).

If we would decide to host guides internally on RA, we would need to prepare the tooling for creating and editing guides. It would need more work to make all of that, but we could better utilize our achievement/user/game tags, and also create relations automatically (like showing on user profile all guides written by them).

Despite of the way we would choose, there would be some work to do (like database changes and some guide management tools).

**Comment by Jamiras** [comment]
#1408 added the sidebar link for the guide. Leaving this open for potentially replacing the github solution with an on-site solution.

See also #662
